cms/templates/certificates.html

```diff
@@ -92,7 +92,7 @@ <h3 class="title-3">${_("Working with Certificates")}</h3>
             <p>${_("To view a sample certificate, choose a course mode and select {em_start}Preview Certificate{em_end}.").format(em_start='<strong>', em_end="</strong>")}</p>
             
             <h3 class="title-3">${_("Issuing Certificates to Learners")}</h3>
-            <p>${_("To begin issuing certificates, a course team member with the Admin role selects {em_start}Activate{em_end}. Course team members without the Admin role cannot edit or delete an activated certificate.").format(em_start='<strong>', em_end="</strong>")}</p>
+            <p>${_("After you have configured and previewed your certificate, ask your edX partner manager to {em_start}activate{em_end} your certificate. When your certificate has been activated, you can begin {em_start}issuing{em_end} certificates to learners. Course team members without the Admin role cannot edit or delete an activated certificate.").format(em_start='<strong>', em_end="</strong>")}</p>
```

Comment:
@mehulsr This looks great! I would remove the ```{em_start}``` and ```{em_end}``` tagging around "activate" and "issuing". (We only mark terms that are actual UI labels bold.)
Have you tested this on a devstack instance or sandbox?